### PR TITLE
CFE-2406: cf-execd now re-parses augments on policy reload

### PR DESCRIPTION
**Pull request by nickanderson**
Changelog: Title
(cherry picked from commit cdb25fd7bbe85e07cf60bf494e08ac5f4751185a)

### CODE REVIEW

**Comment by nickanderson** [comment]
@Lex-2008 @jimis this change passed in master, failing here on some nfs test ideas?

**Comment by nickanderson** [comment]
FWIW seems core on 3.10.x is failing to build in the same way here https://github.com/cfengine/core/pull/2919 ?

File: cf-execd/cf-execd.c [hunk 1]
```diff
@@ -623,6 +623,9 @@ static bool ScheduleRun(EvalContext *ctx, Policy **policy, GenericAgentConfig *c
 
         DetectEnvironment(ctx);
 
+        Log(LOG_LEVEL_INFO, "Re-evaluating augments" );
```
Comment (jimis) [inline cf-execd/cf-execd.c]: VERBOSE please, no need to spam syslog with this.

Comment (nickanderson) [reply cf-execd/cf-execd.c]: You said it was OK as is. So merging.

**Pull request merged**